Info.plist

```diff
@@ -24,18 +26,42 @@
 	<dict>
 		<key>tsInputModeListKey</key>
 		<dict>
-			<key>im.rime.inputmethod.Squirrel</key>
+			<key>im.rime.inputmethod.Squirrel.Hans</key>
 			<dict>
-				<key>TISInputSourceID</key>
-				<string>im.rime.inputmethod.Squirrel.Rime</string>
 				<key>TISIntendedLanguage</key>
-				<string>zh</string>
+				<string>zh-Hans</string>
 				<key>tsInputModeAlternateMenuIconFileKey</key>
 				<string>rime.pdf</string>
 				<key>tsInputModeCharacterRepertoireKey</key>
 				<array>
-					<string>zh-Hans</string>
-					<string>zh-Hant</string>
+					<string>Hans</string>
+					<string>Han</string>
+				</array>
+				<key>tsInputModeDefaultStateKey</key>
+				<true/>
+				<key>tsInputModeIsVisibleKey</key>
+				<true/>
+				<key>tsInputModeKeyEquivalentModifiersKey</key>
+				<integer>4608</integer>
+				<key>tsInputModeMenuIconFileKey</key>
+				<string>rime.pdf</string>
+				<key>tsInputModePaletteIconFileKey</key>
+				<string>rime.pdf</string>
+				<key>tsInputModePrimaryInScriptKey</key>
+				<true/>
+				<key>tsInputModeScriptKey</key>
+				<string>smUnicodeScript</string>
+			</dict>
+			<key>im.rime.inputmethod.Squirrel.Hant</key>
```

Comment:
I'm very confused. Is this an input mode, conceptually?

For your reference, mozc defined one input source per input mode:
https://github.com/google/mozc/blob/master/src/mac/Info.plist

Reply:
This is an input mode stating RIME's instance under the region of zh-Hant.

I know nothing about mozc. Wherever TISInputSourceID gets put, this PR will require existing users to reenable RIME, since the tsInputModeListKey of the instances are changed.

P.S.: 我沒分析過 mozc。萬一 mozc 的多副本是用諸如衛星應用等形式做背後支持的話，那將 TISInputSourceID 塞到 tsInputModeListKey 下方就還能說得過去。如果你實在不放心的話，要不要改天我問問 Apple Japan 的 mzp 老師？或者你電郵問一下 Apple Developer Relations？（Apple Developer Relations 應該更負責一些，畢竟你交了會員費。）

Reply:
另外，input mode 差分化了之後，可以在輸入法的 input controller 內依據當前不同的 input mode 做出不同的準備行為（比如威注音就是載入不同的簡繁辭庫。）

Reply:
Apple的代碼文檔說得比較清楚。InputMode和總體的InputMethod都要定義TISInputSourceID，也都可以缺省定義。